content/graphql/quick-start.md

```diff
@@ -58,6 +58,8 @@ export class AppModule {}
 
 As mentioned, these options will be forwarded to the `ApolloServer` constructor.
 
+> warning **Warning** If you use the `apollo-server-fastify` package with multiple GraphQL endpoints in a single application, enable the `disableHealthCheck` setting to` true` in the configuration` GraphQLModule`.
+
```

Comment:
If this suggestion applies only to the case when a developer has multiple endpoints, "Getting started with GraphQL & TypeScript" is not the right place for this hint

Reply:
I agree, I had put it there as there is the example of the `GraphQLModule` configuration but it is not very sure it was really good. 

If we move it to the section dedicated to `Multiple endpoints`?

Reply:
Sounds good

Reply:
Ok, moved it in `Multiple endpoints`